Sources/Views/PlayButtonView.swift

```diff
@@ -28,7 +28,7 @@ open class PlayButtonView: UIView {
 
     // MARK: - Properties
 
-    open let triangleView = UIView()
+    public let triangleView = UIView()
```

Comment:
Emm, how about make it `open` and `var`?

Reply:
What would be the use case for allowing someone to set a new value?

Reply:
Maybe users want to do some custom things? Make it `open` have no side-effect, right?

Reply:
Before it was open but a let. So technically they can't change it. But if it's changed to var someone can set it to something. But if they set it that won't automatically add the new view to the superview or lay it out.